docs-chef-io/content/server/server_monitor.md

```diff
@@ -223,6 +223,10 @@ by Chef support services.
 
 ### Redis
 
+Note: From Chef Infra Server 15.10.x onwards, KeyDB service replaces Redis.
```

Comment:
**Replace it with:** {{< note >}} From Chef Infra Server 15.10. x onwards, the [KeyDB] (https://docs.keydb.dev/docs/about) service replaces Redis. The configuration name and service name will continue to be `redis_lb`. {{< note/ >}}